imfilebrowser.h

```diff
@@ -90,8 +90,24 @@ namespace ImGui
         // default value is 0 (the first type filter)
         void SetCurrentTypeFilterIndex(int index);
 
+        // when ImGuiFileBrowserFlags_EnterNewFilename is set
+        // this function will pre-fill the input dialog with a filename.
+        void SetInputName(std::basic_string<char> input) noexcept
```

Comment:
this function doesn't take ownership of the input string, so using `std::string_view` can avoid an unnecessary string construction/copy. besides, using `std::basic_string` instead of `std::string` is quite uncommon.

Reply:
Fixed that, included string-view